scipy/integrate/tests/test_quadpack.py

```diff
@@ -31,21 +25,24 @@ def assert_quad(value_and_err, tabled_value, errTol=1.5e-8):
 
 
 class TestCtypesQuad(TestCase):
-    @dec.skipif(_ctypes_missing, msg="Ctypes library could not be found")
     def setUp(self):
         if sys.platform == 'win32':
             if sys.version_info < (3, 5):
-                file = ctypes.util.find_msvcrt()
+                files = [ctypes.util.find_msvcrt()]
```

Comment:
For the record, to avoid such murking around trying to find `libm`, you could compile your own wrapper function simply calling `sin()` (that's what we do in Numba).